webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -6281,6 +6398,14 @@ const firstVisibleMsg = {
                         userStatus: 4,
                     });
                 }
+
+                const businessHourBox = document.getElementById(
```

Comment:
duplicate code, write code once and reuse it wherever requqired

Reply:
fixed, removed duplicity in code